pom.xml

```diff
@@ -42,6 +42,7 @@
             <groupId>org.ow2.asm</groupId>
             <artifactId>asm</artifactId>
             <version>7.2</version>
+            <scope>provided</scope>
```

Comment:
I'm fine with the rest, i just don't get the rational of this change. Why would it be provided?

Reply:
Because I assume that the user of this plugin should have an existing dependency on the ASM.
Then switching the scope to `provided` has a benefit: the user does not need to worry about the dependency confliction.

Reply:
ok, i guess that makes sense,.